.github/ISSUE_TEMPLATE/release-team-lead.md

```diff
@@ -64,6 +64,7 @@ As you work through the checklist, use the following PRs as guides:
   - [kubernetes-sig-leads](https://groups.google.com/forum/#!forum/kubernetes-sig-leads) (Add as Member)
 - [ ] Grant calendar access
 - [ ] Grant Zoom credentials (host key)
+- [ ] Add incoming leads to `release-team-leads` Slack Group [(`kubernetes/community`)](https://github.com/kubernetes/community/blob/master/communication/slack-config/sig-release/usergroups.yaml)
```

Comment:
I don't know if it's worth mentioning, but their slack IDs need to be added to [users.yaml](https://github.com/kubernetes/community/blob/master/communication/slack-config/users.yaml) first.

Reply:
Ah, that is a good thing to note, I think it's worth mentioning. Thanks, I will update!